src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -91,11 +93,19 @@ public static function getSubscribedEvents()
     protected function logException(\Exception $exception, $message)
     {
         if (null !== $this->logger) {
-            if (!$exception instanceof HttpExceptionInterface || $exception->getStatusCode() >= 500) {
-                $this->logger->critical($message, array('exception' => $exception));
-            } else {
-                $this->logger->error($message, array('exception' => $exception));
+            $logLevel = LogLevel::ERROR;
+            if ($exception instanceof HttpExceptionInterface) {
+                $statusCode = $exception->getStatusCode();
+                if (isset(static::$httpStatusCodeLogLevel[$statusCode])) {
+                    $logLevel = static::$httpStatusCodeLogLevel[$statusCode];
```

Comment:
should there be a way to disable a log from happening? i.e. `404: false`

Reply:
I think it's simplest to only have this set logging level. The user can set a 404 to DEBUG, and raise the logging level to control the output